### PR TITLE
Added support for unknown relationship names

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by JudahSchwartz — APPROVED** [review]

File: Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py [hunk 1]
```diff
@@ -544,17 +554,24 @@ def get_mitre_value_from_id(client, args):
...
 
         if attack_pattern_name and len(attack_id) > 5:  # sub-technique
-            parent_name = tc_source.query([
-                Filter("external_references.external_id", "=", attack_id[:5]),
+            parent_objects = tc_source.query([
                 Filter("type", "=", "attack-pattern")
-            ])[0]['name']
+            ])
+            parent_object = list(filter(lambda attack_pattern_obj:
+                                 filter_attack_pattern_object_by_attack_id(attack_id[:5],
```
Comment (JudahSchwartz) [inline Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py:571]: why :5? lets comment the reason/ extract 5 to a constant with a good name

File: Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py [hunk 2]
```diff
@@ -536,6 +535,15 @@ def attack_pattern_reputation_command(client, args):
     return command_results
 
 
+def filter_attack_pattern_object_by_attack_id(attack_id, attack_pattern_object):
```
Comment (JudahSchwartz) [inline Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py:538]: add a docstring

**Review by JudahSchwartz — APPROVED** [review]
nice`1 Couple documentation questions

File: Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py (hunk 1 above)
Comment (MLainer1) [reply Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py:571]: if there is a sub-technique, the attack_id will be `T1070.008` , will add a meaningful name

File: Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py (hunk 2 above)
Comment (MLainer1) [reply Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py:538]: adding

**Pull request merged**